tests/test_diaPipe.py

```diff
@@ -36,7 +36,7 @@
 class TestDiaPipelineTask(unittest.TestCase):
 
     @classmethod
-    def _makeDefaultConfig(cls):
+    def _makeDefaultConfig(cls, doAlerts=False):
```

Comment:
Would you consider using `doSerializeAlerts` here? (Or whatever you change that to, if you think my comment on naming is helpful). Two reasons: one for consistency, and the other because `doAlerts` is not very descriptive (“do alerts what?”).